src/build/partition.jl

```diff
@@ -70,6 +75,29 @@ function _remove_imrg(df::DataFrame, x::Pair{Symbol,Array{String,1}})
 end
 
 
+"""
+This function combines fd values into fdcat values upfront. Doing so changes the results of
+the calibration routine slightly, but isn't too bad. This is the only mapping
+still performed in the build stream. If the difference in output is alright, we can
+move this mapping to the data stream.
+"""
```

Comment:
**For the future**. we can do some more mapping up front, but it will perturb calibration results. (Post-EEM!) I can quantify whether this is worth it and we can discuss whether this is worth it. For now, this is a quick way to do it.